<?xml version="1.0"?>
<launch>

  <arg name="prefix"	default="a_bot_gripper_"/>
  <arg name="device"	default="robotiq_85"/>

  <node name="$(arg prefix)controller"
	pkg="aist_robotiq" type="cmodel_controller.py"
	output="screen" respawn="false">
    <rosparam file="$(dirname)/../config/$(arg device).yaml"
	      subst_value="true"/>
  </node>

</launch>
